client/templates/minutes/minutesEditParticipants.js

```diff
@@ -84,6 +84,16 @@ Template.minutesEditParticipants.helpers({
         }
     },
 
+    switch2MultiColumn() {
+        let aMin = new Minutes(_minutesID);
+
+        if (!Session.get("global.isMobileWidth")) {
```

Comment:
Unfortunately, we cannot get rid of the `switch2MultiColumn`-helper completely because this includes the logic to present the multi-column view based on the amount of participants. Thanks to @RobNeXX for this advice.